### PR TITLE
Update dependencies and format code

### PR DESCRIPTION
**Pull request by lucacorti**
This updates dependencies and formats the code with the Elixir code formatter, which happens to fix a couple compilation warnings about code indentation.

### CODE REVIEW

File: lib/hpack/huffman.ex [hunk 1]
```diff
@@ -24,543 +24,1054 @@ defmodule HPack.Huffman do
...
-  defp dec(<< 0x5::5, rest::bitstring >>, length, result), do: dec(rest, length - 5, result <> <<101>>)
-  defp dec(<< 0x25::6, rest::bitstring >>, length, result), do: dec(rest, length - 6, result <> <<102>>)
-  defp dec(<< 0x26::6, rest::bitstring >>, length, result), do: dec(rest, length - 6, result <> <<103>>)
-  defp dec(<< 0x27::6, rest::bitstring >>, length, result), do: dec(rest, length - 6, result <> <<104>>)
-  defp dec(<< 0x6::5, rest::bitstring >>, length, result), do: dec(rest, length - 5, result <> <<105>>)
-  defp dec(<< 0x74::7, rest::bitstring >>, length, result), do: dec(rest, length - 7, result <> <<106>>)
-  defp dec(<< 0x75::7, rest::bitstring >>, length, result), do: dec(rest, length - 7, result <> <<107>>)
-  defp dec(<< 0x28::6, rest::bitstring >>, length, result), do: dec(rest, length - 6, result <> <<108>>)
-  defp dec(<< 0x29::6, rest::bitstring >>, length, result), do: dec(rest, length - 6, result <> <<109>>)
-  defp dec(<< 0x2a::6, rest::bitstring >>, length, result), do: de
```
Comment (OleMchls) [inline lib/hpack/huffman.ex]: I disagree with this change. This new version is not readable anymore.

File: test/hpack/huffman_test.exs [hunk 2]
```diff
@@ -6,36 +6,64 @@ defmodule HuffmanTest do
...
+             0x7::5,
+             0x14::6,
+             0x78::7,
+             0x7::5,
+             0x2C::6,
+             0x28::6,
+             0x24::6,
+             0x3F8::10,
+             0b111111::6
+           >>
```
Comment (OleMchls) [inline test/hpack/huffman_test.exs]: This is not readable. I'd like the previous style better. This applies to all binary blocks in the specs. I will not comment every block to keep comments to an actionable minimum.

**Comment by lucacorti** [comment]
Hey @OleMchls,

I'll revert the formatter changes as per your request.

WRT the Elixir version upgrade, I didn't want to be toot aggressive. I bumped it mostly because some dependency needed Elixir>=1.4. Onestly, I'm interested in 1.7 (and 1.6 eventually), but that would force any dependent project to use at least that versione. I guess 1.6 could be a sane target?

File: lib/hpack/huffman.ex (hunk 1 above)
Comment (lucacorti) [reply lib/hpack/huffman.ex]: Fixed in 3d8fdae

File: test/hpack/huffman_test.exs (hunk 2 above)
Comment (lucacorti) [reply test/hpack/huffman_test.exs]: Fixed in 3d8fdae

**Comment by OleMchls** [comment]
Again, thanks a lot @lucacorti <3

**Comment by OleMchls** [comment]
published as `1.1.0` on hex :)

**Pull request merged**